script/lib/transpile-packages-with-custom-transpiler-paths.js

```diff
@@ -1,29 +1,36 @@
 'use strict'
 
 const CompileCache = require('../../src/compile-cache')
-const fs = require('fs')
+const fs = require('fs-extra')
 const glob = require('glob')
 const path = require('path')
 
 const CONFIG = require('../config')
+const backupNodeModules = require('./backup-node-modules')
+const runApmInstall = require('./run-apm-install')
+
+require('colors')
```

Comment:
Fly-by comment, maybe use [`chalk`](https://www.npmjs.com/package/chalk) in the future instead of using a module that modifies prototypes (because modules modifying prototypes are prone to conflicts and generally unrecommended)